<launch>
  <node pkg="point_cloud_mapping" type="pcd_to_msg_node" name="pcd_to_msg_node" output="screen" respawn="true"
	args="$(optenv pcd /home/pangerci/work/iros_cad/hokuyo/pfh_denoise/lancastertea-denoise.pcd) $(optenv time_pcd 10)" />
  
  <node pkg="cloud_algos" type="box_estimation_node" name="box_estimation_node" output="screen" respawn="true">
    <remap from="~cloud_pcd" to="cloud_pcd" />
    <param  name="output_box_topic" type="string" value="box_marker"/>
  </node>
</launch>
